src/components/blockquote/README.md

```diff
@@ -14,7 +14,6 @@ npm install --save @ecl/blockquote
 - **"lang"** (string) (default: 'en')
 - **"extra_classes"** (optional) (string) (default: '') Extra classes (space separated)
 - **"picture"** (associative array) (default: {}): Image for the blockquote, following ECL Picture structure
-- **"extra_classes"** (optional) (string) (default: '') Extra classes (space separated)
```

Comment:
isn't it better to remove the other..? :) That way we would have extra_classes and extra_attributes one after the other as usual

Reply:
indeed.